hpx/runtime/threads/detail/scheduled_thread_pool.hpp

```diff
@@ -242,6 +242,11 @@ namespace hpx { namespace threads { namespace detail
 
         std::int64_t get_cumulative_duration(std::size_t, bool);
 
+#if defined(HPX_HAVE_ADAPTIVE_COALESCING_COUNTERS) && defined(HPX_HAVE_THREAD_IDLE_RATES)
```

Comment:
What's the relation between adaptive coalescing and the background thread timings? Why can't I just enable the performance counters without the adaptive coalescing option?

Reply:
@sithhell  The plan is to use these counters to do adaptive parcel coalescing so I used the name adaptive coalescing, but I am open to changing it to something else like HPX_HAVE_BACKGROUND_THREAD_COUNTERS. Let me know if you want me to do that instead.

Reply:
yes, that sounds better. And I guess they are orthogonal to the idle rate counters as well.

Reply:
They are orthogonal, but they internally use the same data sources, iirc,